From 9a18a800011faf018ff49220ae368c673c690bdc Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 17 Jul 2012 10:52:54 +0000
Subject: [PATCH] lbu: check permissions when generating apkovl

We want include files and dirs that have changed permissions.

ref #1241
---
 lbu.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lbu.in b/lbu.in
index 3fa157e..a0e68ed 100644
--- a/lbu.in
+++ b/lbu.in
@@ -278,7 +278,7 @@ If <dirname> nor <filename> is not specified, a package named
 }
 
 _gen_filelist() {
-	apk audit --backup --quiet --recursive
+	apk audit --backup --quiet --recursive --check-permissions
 }
 
 cmd_package() {
-- 
1.7.11.2

